use debian library version of liblz4 rather than the embedded copy
--- a/src/bin/termptysave.c
+++ b/src/bin/termptysave.c
@@ -2,7 +2,7 @@
 #include <Elementary.h>
 #include "termpty.h"
 #include "termptysave.h"
-#include "lz4/lz4.h"
+#include <lz4.h>
 #include <sys/mman.h>
 
 #if defined (__MacOSX__) || (defined (__MACH__) && defined (__APPLE__))
--- a/src/bin/meson.build
+++ b/src/bin/meson.build
@@ -31,7 +31,6 @@ terminology_sources = ['private.h', 'cov
                        'termptygfx.c', 'termptygfx.h',
                        'termptyext.c', 'termptyext.h',
                        'termptysave.c', 'termptysave.h',
-                       'lz4/lz4.c', 'lz4/lz4.h',
                        'md5/md5.c', 'md5/md5.h',
                        'utf8.c', 'utf8.h',
                        'win.c', 'win.h',
@@ -41,6 +40,8 @@ terminology_sources = ['private.h', 'cov
                        'tty_keys.h',
                        'sb.c', 'sb.h']
 
+lz4lib = dependency('liblz4')
+
 tybg_sources = ['tycommon.c', 'tycommon.h', 'tybg.c']
 tyalpha_sources = ['tycommon.c', 'tycommon.h', 'tyalpha.c']
 typop_sources = ['tycommon.c', 'tycommon.h', 'typop.c']
@@ -64,7 +65,7 @@ executable('terminology',
            edj_targets,
            install: true,
            include_directories: config_dir,
-           dependencies: terminology_dependencies)
+           dependencies: [terminology_dependencies, lz4lib])
 executable('tybg',
            tybg_sources,
            install: true,
